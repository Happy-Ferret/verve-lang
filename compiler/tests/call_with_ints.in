(bar 4 5)
