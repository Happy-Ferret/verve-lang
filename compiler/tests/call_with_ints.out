push $5
push $4
push $2
call bar
